pkg/operator/vclib/connection.go

```diff
@@ -116,9 +115,7 @@ func (c *VSphereConfig) GetDefaultDatastore() string {
 
 // ValidateConfig validates if current loaded config is valid.
 func (c *VSphereConfig) ValidateConfig(featureGates featuregates.FeatureGate) error {
-	if len(c.Config.VirtualCenter) > 1 && !featureGates.Enabled(features.FeatureGateVSphereMultiVCenters) {
-		return fmt.Errorf("cloud config must define a single VirtualCenter")
-	}
+	// Future feature gate check logic should go here.  Currently empty due to removal of multi vcenter feature gate.
 	return nil
```

Comment:
lets remove the call to `ValidateConfig` and remove the function if it is not doing anything.